src/components/ProfileCard/ProfileCard.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import Avatar from '@material-ui/core/Avatar';
+import IconButton from '@material-ui/core/IconButton';
+import OpenIcon from '@material-ui/icons/Launch';
+import Dialog from 'components/Dialog';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+
+import { formatDate } from 'utils/format';
+import { TeamMember } from 'types';
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        medium: {
+            width: theme.spacing(10),
+            height: theme.spacing(10),
+        },
+        large: {
+            width: theme.spacing(22),
+            height: theme.spacing(22),
+        },
+        card: {
+            maxWidth: 300,
+            padding: 25,
+        },
+        black: {
+            color: theme.palette.common.black,
+        },
+        center: {
+            alignItems: 'center',
+            display: 'flex',
+            justifyContent: 'center',
+        },
+        paddingTop: {
+            paddingTop: 30,
+        },
+    })
+);
+
+interface Props {
+    teamMember: TeamMember;
+}
+export default function ProfileCard({ teamMember }: Props) {
+    const classes = useStyles();
+    const [open, setOpen] = React.useState(false);
+
+    const handleClickOpen = () => {
+        setOpen(true);
+    };
+
+    const handleClose = () => {
+        setOpen(false);
+    };
+    return (
+        <Card className={classes.card}>
+            <Grid
+                spacing={2}
+                container
+                direction='column'
+                alignItems='center'
+                justify='center'
+            >
+                <Grid container item justify='flex-end'>
+                    <IconButton
+                        id='openDialogButton'
+                        size='small'
+                        onClick={() => handleClickOpen()}
```

Comment:
Minor optimization. There's an unnecessary function signature here.  Technically, you have an arrow function which calls your `handleClickOpen`, that finally calls `setOpen(true)`.  The arrow function is an unnecessary step, `onClick={handleClickOpen}` would suffice.

Reply:
Oh yeah, that's right!